concourse/scripts/pause-kick-off.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
These two scripts could probably be one script with an argument: `(pause|unpause)`, but I will survive if you leave it as it is.